streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -53,7 +53,6 @@
  * @see KGroupedStream
  * @see KStreamBuilder#stream(String...)
  */
-@SuppressWarnings("unused")
```

Comment:
My IDE identified this line as unnecessary. I guess it doesn't check whether the APIs are used or not. I can put it back if you think it should be there.